CIP-????/README.md

```diff
@@ -0,0 +1,194 @@
+
+Additionally, This specification provides the general guidelines and necessary recommendations for performing secure authenticated web3 requests in the Cardano ecosystem. It covers the two main desired characteristics for a secure payload: It must expire and it must be non-static. Moreover, the signature method proposed in this CIP does not require users to spend funds in a transaction, which further lowers the cost and barriers to entry for users.
+
+Another important aspect for security is how wallets process the payload. They can improve the security using the data inside the payload to warn the users about possible malicious interactions. This specification emphasizes the importance of informing users clearly about the purpose of the payloads and how wallets can use the URI field to apply allow-lists and/or cross-domain policies. It establishes also the requirements and recommendations for server side processing. The server must also ensure the validity of the signature and the payload, as well as of its purpose in order to accomplish the authentication. 
+
+In addition to the aforementioned aspects, this CIP also aims to promote decentralization and enhance security and privacy by enabling users to sign and verify transactions without relying on external servers or third parties. By allowing users to create and sign their own payloads, this specification reduces the dependency on centralized authorities and enhances the security and privacy of the transactions.
+
+### Version history
+
+| Version | Date      | Author                         | Rationale              |   
+|:-------:|-----------|--------------------------------|------------------------|
+| v1      |2022-12-27 | Juan Salvador Magán Valero     | Initial release        |
+
+
+### Reference implementation
+
+* [jmagan/passport-cardano-web3](https://github.com/jmagan/passport-cardano-web3)
+* [jmagan/cardano-express-web3-skeleton](https://github.com/jmagan/cardano-express-web3-skeleton)
+* [jmagan/cardano-nextjs-web3-skeleton](https://github.com/jmagan/cardano-nextjs-web3-skeleton)
+## Path to Active
+
+### Acceptance Criteria
+
+- [X] At least one library should implement this authentication method.
+- [ ] The 80% users should have wallets implementing the following requirements:
+    1. It MUST detect when the payload is formatted using this specification.
+    2. The information contained in the payload MUST be parsed and formatted in the signing pop-up.
+    3. The wallet SHOULD update the timestamp just before the payload is signed.
+    4. The wallet MUST detect if the URI is in the allow list.
+    5. The wallet SHOULD warn the user against cross-domain requests.
```

Comment:
These act as an update to CIP-30.  A wallet that does this is not following the specification defined for CIP-30 for signData which does not include these rules for payload data processing.  There would be no way for a dApp to know if a wallet obeyed CIP-30 or CIP-93 because CIP-93 is not currently defined as an extension of CIP-30.

Reply:
I partially addressed this in the previous comment. The proposed changes do not modify the API, and the signData method retains the same arguments and return value. This CIP involves three key parties: the wallet, the payload, and the server processing. It would be challenging to cover all these aspects within a wallet-specific CIP alone, which is why it is presented as a comprehensive specification covering the entire process.

Reply:
likewise resolved as per https://github.com/cardano-foundation/CIPs/pull/442#discussion_r1235142130.